packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -72,6 +77,11 @@ export interface ITooltipProps extends React.HTMLAttributes<HTMLDivElement | Too
    * Call to provide customized styling that will layer on top of the variant rules.
    */
   styles?: IStyleFunctionOrObject<ITooltipStyleProps, ITooltipStyles>;
+
+  /**
+   * Forces the tooltip to dismiss.
+   */
+  dismiss?: () => void;
```

Comment:
is this requires as a prop?

Reply:
i mean. this is already in the public interface....what does this do as a prop?

Reply:
With dismiss as a prop, TooltipBase can use TooltipHost's function `_hideTooltip` to toggle the tooltip closed. 

I have it in the interface, too, because that is needed to call `TooltipBase._currentVisibleTooltip.dismiss()`, since `_currentVisibleTooltip` is of type `ITooltip`.

Reply:
`TooltipBase._currentVisibleTooltip.dismiss()` is referencing the public interfae in ITooltip, not ITooltipProps.

Reply:
Yep! And then in dismiss() [on line 90](https://github.com/OfficeDev/office-ui-fabric-react/pull/7105/files/45dc8d6739b179ed0ede1688b8e5bdbea3a853f9#diff-00719f3cbd47c1407b8d401275c4292eR92), we call this.props.dismiss(). We pass in `this._hideTooltip` to the dismiss prop [here](https://github.com/OfficeDev/office-ui-fabric-react/pull/7105/files/45dc8d6739b179ed0ede1688b8e5bdbea3a853f9#diff-578e19e31ce4615467278be75612fbd2R98).

Reply:
Oh...so this is just a callback? Do you mean 'onDismiss'? A function to be called in the case that it is dismissed?

Reply:
this dismiss isn't necessary anymore.